build-config/make/sysroot.make

```diff
@@ -52,6 +55,10 @@ $(DEV_SYSROOT_INIT_STAMP): $(TREE_STAMP) | $(XTOOLS_BUILD_STAMP)
 	$(Q) echo "==== Preparing a new development sysroot ===="
 	$(Q) rm -rf $(DEV_SYSROOT)
 	$(Q) cp -a $$($(CROSSBIN)/$(CROSSPREFIX)gcc -print-sysroot) $(DEV_SYSROOT)
+	$(Q) rsync -rl $(XTOOLS_INSTALL_DIR)/$(TARGET)/$(TARGET)/lib $(DEV_SYSROOT)
+       ifeq ($(ARCH),$(filter $(ARCH),arm64 x86_64))
+		$(Q) rsync -rl $(XTOOLS_INSTALL_DIR)/$(TARGET)/$(TARGET)/lib64 $(DEV_SYSROOT)
+       endif
```

Comment:
Thanks for those patches.  That helped.  We are definitely getting closer.

Here are some more machines that need fixing for custom busybox patches:
```
quanta/quanta_common_rangeley
celestica/cel_seastone
celestica/cel_e1031
```

These ones need the CONFIG patch:
```
dell/dell_s4000_c2338
wnc/wnc_sst1_n1
alphanetworks/alphanetworks_snx60a0_486t
wiwynn/wiwynn_andromeda
accton/accton_as6712_32x
accton/accton_as5512_54x
accton/accton_as7412_32x
accton/accton_as6812_32x
accton/accton_as7312_54x
accton/accton_as7716_32x
accton/accton_as7512_32x
accton/accton_as5812_54x
accton/accton_as5812_54t
accton/accton_as5822_54x
accton/accton_as7712_16x
accton/accton_as7712_32x
accton/accton_wedge_16x
```